libbeat/beat/beat.go

```diff
@@ -252,6 +254,11 @@ func (b *Beat) launch(bt Creator) error {
 		return err
 	}
 
+	err = b.loadTemplate()
```

Comment:
Good point. This just go a lot trickier to implement as it now uses its own client.